README.md

```diff
@@ -1 +1,41 @@
-# weather-app
\ No newline at end of file
+# Weather App.
+
+> This project consist on build a webpage where a user can look for the day weather.
+
+
+
+## Built With
+
+- JS
+- Module Pattern
+- Webpack
+- ES6
+- CSS
+
+## Getting Started
+
+- Download the repo con your computer.
+- Drag and drop the index.html file from the dist folder into your web browser.
```

Comment:
### [Optional]
Add the list of step to git clone your repo.
**Kindly update your `README` file and remove unnecessary content**.
Checkout out the below link for help, free free to follow this [template](https://github.com/microverseinc/readme-template) from pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/